segments/np_spotify_mac.script

```diff
@@ -7,17 +7,23 @@ end tell
 
 if process_list contains "Spotify" then
 	tell application "Spotify"
-		if player state is playing then
+		if player state is playing or player state is paused then
 			set track_name to name of current track
 			set artist_name to artist of current track
 			#set album_name to album of current track
 			set trim_length to 40
-			set now_playing to artist_name & " - " & track_name
+			if player state is playing then
+				set now_playing to "► " & artist_name & " - " & track_name
+			else
+				set now_playing to "❙❙ " & artist_name & " - " & track_name
+			end if
 			if length of now_playing is less than trim_length then
 				set now_playing_trim to now_playing
 			else
 				set now_playing_trim to characters 1 thru trim_length of now_playing as string
 			end if
 		end if
 	end tell
+else
```

Comment:
I think that if the music is stopped, then the segment should be hidden (no output here). That's how the other NP segments work.